actor-typed-tests/src/test/scala/org/apache/pekko/actor/typed/scaladsl/DispatcherSelectorSpec.scala

```diff
@@ -16,8 +16,10 @@ package org.apache.pekko.actor.typed.scaladsl
 import com.typesafe.config.Config
 import com.typesafe.config.ConfigFactory
 import org.scalatest.wordspec.AnyWordSpecLike
-
 import org.apache.pekko
+import pekko.actor.typed.DispatcherSelector
```

Comment:
can you reorder the imports and put back the empty line above?

Reply:
Sorry about the delay, could you review this again and contain this in release 1.1.0?